Create ui-c8y-1021-18-0-enable-global-context-for-auto-refresh-6203-graft-release-cd-7739.md (#2706)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-18-0-enable-global-context-for-auto-refresh-6203-graft-release-cd-7739.md

---
date: ""

> title: Auto refresh can now be enabled in any dashboard

Reviewer: ```suggestion
title: Auto refresh can now be enabled globally in dashboards
```

> title: Enable global context for "auto refresh" (#6203) [GRAFT][release/cd]  (#7739)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Auto refresh can now be enabled in any dashboard
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58878
version: 1021.18.0
---

> Enable global context for "auto refresh" (#6203) [GRAFT][release/cd]  (#7739)

Author: The Claude AI suggested to use the following body:
```suggestion
In the past, the auto refresh functionality was only available in certain parts of the application. With this change, auto refresh is now enabled globally across the entire application. This means that data displayed in the application will automatically update in real-time without requiring a manual refresh. The impact of this change is an improved user experience, as users will always see the most up-to-date information without needing to take any additional action.
```

Reviewer 2: This one has to be improved:
```suggestion
Previously, auto refresh was only available in certain parts of the application, such as the alarms view and widgets and map widgets. With this change, auto refresh can now be enabled globally in any dashboard. This means that the data displayed in widgets, that support and enabled this functionality, will be automatically updated at set and common intervals, without the need to manually refresh or set their intervals separately. The impact of this change is an improved user experience as users will always see the most up-to-date information without having to take any additional action.
```

> Previously, auto refresh was only available in certain parts of the application, such as the **Alarms** page, "Alarms" widget and "Map" widget. With this change, auto refresh can now be enabled globally in any dashboard. This means that the data displayed in widgets, that support this functionality and have it enabled, will be automatically updated at set and common intervals, without the need to manually refresh or set their intervals separately. The impact of this change is an improved user experience as users will always see the most up-to-date information without having to take any additional action.

Reviewer: @luja-c8y Is that what you meant? such as the **Alarms** page, "Alarms" widget and "Map" widget

can now be enabled globally in any dashboard > Shouldn´t we remove "globally" as this is not really the case? Or can it also be enabled globally for all dashboards?

Reviewer 2: @BeateRixen "global" might be misleading, but, imho, you explained it well, because "globally" is used here in the context of a dashboard.

Overall it is a [dashboard time range](https://cumulocity.com/docs/cockpit/working-with-dashboards/#global-time-context) but instead of using real-time we pull data for all widgets that have this feature enabled in each widget configuration (currently it is: alarms and map widgets, but soon data points table widget will be added to this list too).

I only mentioned the **alarms page** as an example where we use pulling instead of real-time, but the feature described here **does not** affect the **alarms page**.

Reviewer: Thanks for the explanation, @luja-c8y. Sounds good to me. Please approve.

> Previously, auto refresh was only available in certain parts of the application, such as the alarms view and widgets and map widgets. With this change, auto refresh can now be enabled globally in any dashboard. This means that the data displayed in widgets, that support and enabled this functionality, will be automatically updated at set and common intervals, without the need to manually refresh or set their intervals separately. The impact of this change is an improved user experience as users will always see the most up-to-date information without having to take any additional action.

Reviewer: ```suggestion
Previously, auto refresh was only available in certain parts of the application, such as the **Alarms** page, "Alarms" widget and "Map" widget. With this change, auto refresh can now be enabled globally in any dashboard. This means that the data displayed in widgets, that support this functionality and have it enabled, will be automatically updated at set and common intervals, without the need to manually refresh or set their intervals separately. The impact of this change is an improved user experience as users will always see the most up-to-date information without having to take any additional action.
```
